### ISSUE TITLE
[Feature Request][Dashboard] Add logs to Dashboard

### ISSUE THREAD

**Issue by HuangDaHui** [allburov/whatsapp-http-api#646]
This is just a suggestion from me. If possible, it will be much easier when we need to provide logs to you.

**Comment by devlikepro** [comment]
Yes! Now you can observer events using [Event Monitor](https://waha.devlike.pro/docs/how-to/waha-dashboard/#event-monitor), but we're working on proper logs solution too!

it'll work on for default "docker-compose" installation tho, I don't think we'll add more complex scenarios for that - if one have k8s or portainer you probably already have something like Grafana or other logs solutions

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by HuangDaHui** [comment]
> Yes! Now you can observer events using [Event Monitor](https://waha.devlike.pro/docs/how-to/waha-dashboard/#event-monitor), but we're working on proper logs solution too!
> 
> it'll work on for default "docker-compose" installation tho, I don't think we'll add more complex scenarios for that - if one have k8s or portainer you probably already have something like Grafana or other logs solutions
> 
> [![patron:PRO](https://camo.githubusercontent.com/4f70c3a9ee2450c288faea5e1a1b7ebd0ffc1571450a59676b300ffb32b32537/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

OK, thank you
Looking forward to your updates
